pkgs/desktops/gnome/core/gnome-remote-desktop/default.nix

```diff
@@ -20,12 +20,18 @@
 , libopus
 , libxkbcommon
 , gdk-pixbuf
+, cmake
```

Comment:
Are you sure `cmake` is needed? Meson supports it for finding dependencies but typically, `pkg-config` will be sufficient. See <https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/meson-cmake.md> for more info.

Reply:
It must've been included (or superseded) by another dependency, because the build used to fail because it was missing when I removed `"-Dtests=false"`. Removing…